### ISSUE TITLE
[BUG] new tab adress bar show 'chrome-search://local-ntp/local-ntp.html'

### ISSUE THREAD

**Issue by gnadelwartz** [kiwibrowser/src.next#288]
**Describe the bug**
If you select the adress bar of a new tab with feauture 'Keep content when focusing adress bar' enabled the following is displayed 'chrome-search://local-ntp/local-ntp.html' in the adress bar

**To Reproduce**
enable 'Keep content when focusing adress bar', popen new tab and klick in adress bar

**Expected behavior**
empty adress bar

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Chrome Book
 - Android Version : Chrome OS stating as Android 9 
 - Kiwi Browser Version: 93.0.4577.22

**Additional context**
Seems not to happen on my Redmi Note 9 pro with Android 11

**Comment by kiwibrowser** [comment]
I agree with you, while it's technically logical, it doesn't make sense, the scenario where you have to copy-paste the homepage URL is so rare compared to the amount of times you may need to enter a keyword

The issue is here: https://github.com/kiwibrowser/src.next/blob/258c95b5c85c8b806f33d0aaac5b241a2b9e975f/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/editurl/EditUrlSuggestionProcessor.java#L94

The condition should probably be something like:

if (ContextUtils.getAppSharedPreferences().getBoolean("keep_address_bar_content", false) && !addressbar_content_doesnt_starts_with_chrome-search)

**Comment by gnadelwartz** [comment]
didn't notice that its the internal homepage URL and was wondering why network time protocol is relevant for a new tab ;-)

**Comment by DUOLabs333** [comment]
For the `!addressbar_content_doesnt_starts_with_chrome-search` part, this might work `!activeTab.getUrl().startsWith("chrome-search://")`

**Comment by kiwibrowser** [comment]
Absolutely @DUOLabs333 -- and I am almost ready with the build system for forks (contributor builds) so if someone in few days (not all is ready) wants to play and use this issue to test the system it can be good

**Comment by DUOLabs333** [comment]
I'll be willing to try it out, if there was a way to back it up: the only current way is to have root and manually extract the files. Maybe an option for Export/Import? I'm not too good at Java, but I can try.

**Comment by kiwibrowser** [comment]
It's a completely separate app when you do a change, no worry, so your data is safe.
Give me a few days, and you'll be surprised how easy it is to try :)

**Comment by kiwibrowser** [comment]
If you are interested to try your build without waiting it's here: https://build.find.kiwi/apks/1118391985/out.apk (it's not called Kiwi at this stage, as it's during testing).
It will install separately

Thanks a lot for the fix!

**Comment by kiwibrowser** [comment]
(and I tested it works!)

**Comment by DUOLabs333** [comment]
@kiwibrowser Is there a way to import from the "Play Store" version to the "GitHub" version, especially since they have different package names.

**Comment by kiwibrowser** [comment]
> @kiwibrowser Is there a way to import from the "Play Store" version to the "GitHub" version, especially since they have different package names.

No, but one goal is to make the data more easy to extract (bookmarks are already ok) and I feel this is the right solution

**Comment by gnadelwartz** [comment]
@DUOLabs333 thanks for the fix and @kiwibrowser to include it in Kiwi.

Hopefully there will be an updated, much improved Kiwi version availible in Play Store soon :-)
